n2y/plugins/mermaid.py

```diff
@@ -32,10 +39,14 @@ def __init__(self, client, page, notion_data, get_children=True):
     def to_pandoc(self):
         temp_fd, temp_filepath = tempfile.mkstemp(suffix=".png")
         os.close(temp_fd)
+        temp_config_fd, temp_config_filepath = tempfile.mkstemp(suffix=".json")
```

Comment:
What is the benefit of generating the config file on the fly, vs just including a config file in the python source code?

Reply:
Hm, good question. I think the main benefit would be that it'd be easier to add more config parameters/have dynamic  parameters (derived from cli arguments perhaps) if it's defined as a python dict like this.

Reply:
That makes sense and seems like a good argument to me!